### DISCUSSION TITLE
Different Server Version (365306)

### DISCUSSION THREAD

**Discussion by Majobaga** [wolveix/satisfactory-server#358] [Q&A]
Hello, I'm trying to start this Satisfactory Server as a container in Unraid and I'm running into an issue with the Server Version where it won't let me join my self hosted server because of a mismatch. The question I have is if there's a way to run this container but edit the server version back to 365306. Thanks!

![image](https://github.com/user-attachments/assets/36e910b8-3569-4f99-bc89-74f9135d91e0)

**Comment by wolveix** [comment]
Hello, no the container doesn't support this. You need to update your client to the latest version, or set the container's `SKIPUPDATE` environment variable to disable future updates.
